ztp/siteconfig-generator/siteConfig/siteConfigHelper.go

```diff
@@ -0,0 +1,44 @@
+package siteConfig
+
+import (
+	"log"
+
+	yaml "gopkg.in/yaml.v3"
+)
+
+// Add ztp deploy annotation to all siteconfig generated CRs
+// A dynamic value might be added later
+func addZTPAnnotation(clusterCRs []interface{}) ([]interface{}, error) {
+
+	for _, v := range clusterCRs {
+		cr := v.(map[string]interface{})
+		metadata := cr["metadata"].(map[string]interface{})
+		annotations, _ := metadata["annotations"].(map[string]interface{})
+		annotations[ztpAnnotation] = nil
```

Comment:
In the testdata below, it looks like we're a bit inconsistent - We have
```
ran.openshift.io/ztp-gitops-generated: null
```
in some places and 
```
ran.openshift.io/ztp-gitops-generated: ""
```
in others.  I'd vote for using the empty string instead of nil everywhere, please!

Or maybe even:
```
ran.openshift.io/ztp-gitops-generated: "true"
```
Or
```
ran.openshift.io/ztp-gitops-generated: "{}"
```

Not sure what a good starting value is, if we want to eventually have the ability for it to be structured data?

Reply:
It has been changed to 'ran.openshift.io/ztp-gitops-generated: "{}"'.